tests/python/contrib/test_tensorrt_int8_exp.py

```diff
@@ -0,0 +1,129 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# FP16 TRT command to run : TVM_TENSORRT_USE_FP16=1 python test_trt.py
+# INT8 TRT command to run : TVM_TENSORRT_USE_INT8=1 TENSORRT_NUM_CALI_INT8=10 python test_trt.py
```

Comment:
Let's following the format like other UTs to use:
```python

def test_xxxxx():
    # You can use `os.environ["TVM_TENSORRT_USE_INT8"]=1` to set environment variable here
    # your test code here

if __name__ == "__main__":
    pytest.main([__file__])
```

Reply:
Hi @jcf94 I think ci should be passed, please review it